Formula/mecab-ipadic-neologd.rb

```diff
@@ -0,0 +1,34 @@
+class MecabIpadicNeologd < Formula
+  desc "Neologism dictionary for MeCab"
+  homepage "https://github.com/neologd/mecab-ipadic-neologd"
+  url "https://github.com/neologd/mecab-ipadic-neologd/archive/b815a08d0478714007856d53f13551c92e9b0b5c.tar.gz"
```

Comment:
Unfortunately, upstream at https://github.com/neologd/mecab-ipadic-neologd doesn't do tagged releases, they just commit some new data once in a while (https://github.com/neologd/mecab-ipadic-neologd/commits/master).